Antidote/CallsManager.m

```diff
@@ -271,6 +275,43 @@ - (void)ringingCallDeclineButtonPressed:(RingingCallViewController *)controller
                             error:nil];
 }
 
+#pragma mark - ActiveCallViewControllerDataSource
+
+- (NSInteger)activeCallControllerNumberOfPausedCalls:(ActiveCallViewController *)controller
+{
+    return [[self.allPausedCallsController fetchedObjects] count];
+}
+
+- (NSString *)activeCallController:(ActiveCallViewController *)controller pausedCallerNicknameForCallAtIndex:(NSIndexPath *)indexPath
+{
+    OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];
+
+    OCTFriend *friend = [call.chat.friends firstObject];
+
+    return friend.nickname;
+}
+
+- (NSTimeInterval)activeCallController:(ActiveCallViewController *)controller pauseTimeDurationForCallAtIndex:(NSIndexPath *)indexPath
+{
+    OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];
+
+    return call.callDuration;
```

Comment:
There should be not `callDuration`, but `on hold duration`. So what I suggest to you is to store `on hold start time` and pass it to controller. Controller will have timer that will tick every second, which will be used for updating pause cells.